### PR TITLE
{bio}[intel/2017b] Redundans v0.13c, LAST v914, SNAP v1.0beta.23, GapCloser 1.12-r6, pyScaf 0.12a4, FastIndex 0.11rc7 (+ deps)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 6 out of 8 (8 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/11c2c3d1f4bcd1fcdbe1e085b77793fd for a full test report.

File: easybuild/easyconfigs/f/FastaIndex/FastaIndex-0.11rc7-intel-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('Python', '2.7.14')]
+
+sanity_check_paths = {
+    'files': ['bin/FastaIndex', 'bin/fasta_stats'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+options = {'modulename': name}
```
Comment (verdurin) [inline easybuild/easyconfigs/f/FastaIndex/FastaIndex-0.11rc7-intel-2017b-Python-2.7.14.eb]: @boegel - isn't this the default anyway?

File: easybuild/easyconfigs/p/pyScaf/pyScaf-0.12a4-intel-2017b-Python-2.7.14.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    ('Python', '2.7.14'),
+    ('FastaIndex', '0.11rc7', versionsuffix),
+]
+
+sanity_check_paths = {
+    'files': ['bin/pyScaf'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+options = {'modulename': name}
```
Comment (verdurin) [inline easybuild/easyconfigs/p/pyScaf/pyScaf-0.12a4-intel-2017b-Python-2.7.14.eb]: Same here - why is this required?

File: easybuild/easyconfigs/f/FastaIndex/FastaIndex-0.11rc7-intel-2017b-Python-2.7.14.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/f/FastaIndex/FastaIndex-0.11rc7-intel-2017b-Python-2.7.14.eb]: The default is to use `name.lower()`, since lowercase Python package names are way more common, these are exceptions.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (8 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/7e6bb194078afd0dc584e4e6e42a00a6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2548.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/262aebe3016dbfcba63dacedf26f2373 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/35bbab2538d2088cc9b835873ea79a91 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/75c217069fbc135be6dc014963a25a3a for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!
